ui/src/app/utils/leo-runtime-initializer.tsx

```diff
@@ -68,7 +69,7 @@ export interface LeoRuntimeInitializerOptions {
   workspaceNamespace: string;
   // Callback which is called every time the runtime updates its status. When no runtime is found,
   // the callback is called with a null value.
-  onStatusUpdate?: (RuntimeStatus?) => void;
+  onPollCycleComplete?: (Runtime?) => void;
```

Comment:
nit: I'd prefer `onUpdate` or `onPoll`. Including the word `complete` indicates to me some degree of finality, as in this only runs once the poll cycle has fully complete.

That said, perhaps this change is no longer necessary given the approach we reached during the mobbing session - and is implemented in Pete's PR: https://github.com/all-of-us/workbench/pull/4100/files#diff-ebc3f186ce4c82a34bb3588490435f21R154

Reply:
Yeah, I'll expect that after rebase I'll have to change my branch significantly; I'll see what shakes out afterward